csvcubed/tests/unit/utils/test_pandas.py

```diff
@@ -14,7 +14,7 @@ def test_na_filter_works():
     running the 'read_csv' command and checking that some of the default na-values
     don't get transformed to NaN values.
     """
-    df = read_csv(csv_path)
+    df, _data_errors = read_csv(csv_path)
```

Comment:
Do we have a unit test to ensure that we're returning an error when there are duplicate column titles?